Modules/posixmodule.c

```diff
@@ -5467,6 +5467,49 @@ os__path_islink_impl(PyObject *module, PyObject *path)
 #endif /* MS_WINDOWS */
 
 
+/*[clinic input]
+os._path_splitroot_ex
+
+    path: unicode
+
+[clinic start generated code]*/
+
+static PyObject *
+os__path_splitroot_ex_impl(PyObject *module, PyObject *path)
```

Comment:
Maybe rename the old function `_path_splitroot_win` since it's based on WinAPI `PathCchSkipRoot()` and is only used on Windows in "Lib/importlib/\_bootstrap\_external.py". Then this new function can simply be named `_path_splitroot`, and we won't have to needlessly use an "ex" name.

Reply:
I initially wanted to rename it to `_path_splitanchor`, but @zooba didn't agree:

> But if we're going to leave the old one there, leave it with its original name and make the new function a new function.

I also don't like the "ex" name.

Reply:
OK, with 3 people on board an 1 against I'll rename the functions.

Reply:
Steve's opinion is a very big deal to me, considering how much he's done for Python on Windows for several years now. If he prefers the "ex" function, I'm not going to argue over something that trivial.

Reply:
OK, we keep the "ex" prefix then.